packages/jp2a/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/cslarsen/jp2a
+TERMUX_PKG_DESCRIPTION="a simple JPEG to ASCII converter."
```

Comment:
What this is doing here ?

Reply:
This package just for conversion.
jpeg file to ascii art

Reply:
Just for fun :D